README.md

```diff
@@ -3,40 +3,54 @@
 
 What is this?
 =============
-This is the code for the RECAP Chrome and Firefox extensions, a program that is used to
-liberate millions of documents from the PACER system. To install this extension
-please visit its homepage at https://free.law/recap/.
+This is the code for the RECAP Chrome and Firefox extensions, a program that is
+used to liberate millions of documents from the PACER system. To install this
+extension please visit its homepage at https://free.law/recap/.
 
 
 Reporting Issues
 ================
-Please report all issues at the centralized RECAP issue repo, [/freelawproject/recap](https://github.com/freelawproject/recap/issues).
+Please report all issues at the centralized RECAP issue repo,
+[/freelawproject/recap](https://github.com/freelawproject/recap/issues).
 
 
 Joining as a Developer
 ======================
-If you wish to contribute to our efforts to drain PACER, please get in touch using the [contact form on Free Law Project's website][contact] or simply get to work on an issue that interests you.
+If you wish to contribute to our efforts to drain PACER, please get in touch
+using the [contact form on Free Law Project's website][contact] or simply get to
+work on an issue that interests you.
 
 
 Tips
 ----
-1. You can load an "unpacked extension" from [chrome://extensions/][c] if you enable developer options in Chrome. The same [can be accomplished in Firefox][tempff] by going to [about:debugging][abtdbg], clicking "Load Temporary Add-on" and then selecting any file.
-
-1. While it's true that every court has their own customized version of PACER, there is [a PACER training site that does not charge fees][trainwreck]. You can use this if you wish to work on the system without accruing charges. If you are accruing charges while working on this extension, Free Law Project may be able to help. Let us know.
-
-1. While you're testing things across versions of PACER, you can use a handy website ([json][csv-json]/[html][csv-html]) created by [@audiodude][ad] to find suitable versions of PACER.
-
-1. If you want to auto-zip your code on changes, you have two options. First, you can install a utility called `entr` and run:
+1. You can load an "unpacked extension" from [chrome://extensions/][c] if you
+   enable developer options in Chrome. The same [can be accomplished in
+   Firefox][tempff] by going to [about:debugging][abtdbg], clicking "Load
+   Temporary Add-on" and then selecting any file.
+
+1. While it's true that every court has their own customized version of PACER,
+   there is [a PACER training site that does not charge fees][trainwreck]. You
+   can use this if you wish to work on the system without accruing charges. Note
+   that the extension is disabled on the PACER training site to avoid training
+   documents being sent in. To enable the extension you'll need to edit the
+   blacklist. If you are accruing charges while working on this extension, Free
```

Comment:
I added: `Note that the extension is disabled on the PACER training site to avoid training documents being sent in. To enable the extension you'll need to edit the blacklist.`